Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/KUL9-sts-App-replica-scaleup-jiva/app-replica-scaleup

```diff
@@ -0,0 +1,118 @@
+bash Openshift-EE/utils/e2e-cr jobname:app-replica-scaleup jobphase:Running
+
+echo "*******Deploying Busybox-sts Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-provision-app-scale-jiva metadata="") #change
+echo $test_name
+
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+cp apps/busybox/deployers/run_litmus_test.yml deploy_run_test.yml
+sed -i -e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' \
+-e 's/value: app-busybox-ns/value: app-scale-jiva/g' \  #namespace of application
+-e 's/app: busybox-litmus/app: busybox-litmus-jiva/g' \ #litmus_job_label
+-e 's/value: '\''app=busybox-sts'\''/value: '\''app=busybox-scale-jiva'\''/g' \ #application label
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' deploy_run_test.yml
+cat deploy_run_test.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-litmus-jiva' job=deploy_run_test.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-replica-scaleup-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id KUL9 3-functional "Scale the statefulset application replicas and verify the ring configuration" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "*******Scaling Application replicas**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=scale-app-replicas-jiva metadata="")
+echo $test_name
+
+cd litmus
+cp apps/cassandra/functional/scale_replicas/run_litmus_test.yml run_scaleup_test.yml
+
+sed -i -e 's/app: cassandra-scaleup-litmus/app: scaleup-litmus-jiva/g' \  #application label
+-e 's/value: '\''app=cassandra'\''/value: '\''app=busybox-scale-jiva'\''/g' \ #litmus job label
+-e 's/value: app-cass-ns/value:app-scale-jiva/g' run_scaleup_test.yml
+
+echo "Running the litmus test for Busybox Deployment Scaleup.."
+cat run_scaleup_test.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-scale-jiva' job=run_scaleup_test.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-replica-scaleup-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id KUL9 3-functional "Scale the statefulset application replicas and verify the ring configuration" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "********Deprovisioning Busybox Application*******"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-deprovision-app-scale-jiva metadata="")
+echo $test_name
+
+cd litmus
+cp apps/busybox/deployers/run_litmus_test.yml deprovision_scaleup.yml
```

Comment:
@shashank855 use a different name for the file to copy the content of run_litmus_test. This is to ensure that cstor and jiva test has to run in parallel.